build.gradle

```diff
@@ -47,7 +46,7 @@ if (System.env.GIT_BRANCH) {
 }
 
 // Google's jib task
-jib.to.image = "eoscsynergy/${project.name}:${version}"
+jib.to.image = "davrodgon/${project.name}:${version}"
```

Comment:
@davrodgon Maybe this should be corrected to eoscsynergy for the merge

Reply:
Yes, sorry, I was using that for testing.

Reply:
Changed